src/sentry/options/manager.py

```diff
@@ -272,7 +282,10 @@ def default():
 
         settings.SENTRY_DEFAULT_OPTIONS[key] = default_value
 
-        self.registry[key] = self.make_key(key, default, type, flags, ttl, grace, grouping_info)
+        # TODO: Figure out indirection to allow "legacy" to reference Enums in Models
+        self.registry[key] = self.make_key(
+            key, default, type, flags, ttl, grace, grouping_info, "legacy"
+        )
```

Comment:
Tbh we could just have the manager and model in the same file. Otherwise, you could also just import it in this function instead of the top level.